src/modules/views/analytics/GeneralStatistics.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { AccessTime, People, RunCircleSharp, Save } from '@mui/icons-material';
+import { Box, Grid, Typography } from '@mui/material';
+
+import sumBy from 'lodash.sumby';
+
+import {
+  STATISTIC_RUNNING_VERSIONS_KEY,
+  STATISTIC_SAVED_VERSIONS_KEY,
+  STATISTIC_TIME_SPENT_KEY,
+  STATISTIC_TOTAL_USERS_KEY,
+} from '@/config/selectors';
+import { GeneralMemberStatistic } from '@/interfaces/analytics';
+import { formatSeconds } from '@/utils/chart';
+
+import StatisticCard from './StatisticCard';
+
+const AccessTimeIcon = <AccessTime fontSize="large" color="primary" />;
+const PeopleIcon = <People fontSize="large" color="primary" />;
+const SaveIcon = <Save fontSize="large" color="primary" />;
+const RunIcon = <RunCircleSharp fontSize="large" color="primary" />;
+
+type Props = {
+  generalStatistics: GeneralMemberStatistic[];
+};
+
+const GeneralStatistics = ({ generalStatistics }: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const savedVersions = sumBy(generalStatistics, 'savedVersions');
+  const runningVersions = sumBy(generalStatistics, 'runningVersions');
+
+  const averageTime = generalStatistics.reduce(
+    (acc, curr) => acc + curr.spentTimeInSeconds,
+    0,
+  );
+
+  const { hours, minutes } = formatSeconds(
+    averageTime / generalStatistics.length,
+  );
+  return (
+    <Box marginY={4}>
+      <Typography variant="h6" align="center">
+        {t('General Statistics')}
+      </Typography>
+      <Grid container spacing={2} marginTop={1} justifyContent="center">
+        <StatisticCard
+          icon={AccessTimeIcon}
+          title={t('Average Time Spent By Users')}
+          stat={`${hours || 0} ${t('hours')}, ${minutes || 0} ${t('minutes')}`}
```

Comment:
Same comment here, use only a single translations string with options, or use the internationalisation that is included in date-fns. Using a single string allows for example to have a different format in different languages.